tests/utils/utils.go

```diff
@@ -85,6 +86,52 @@ func DeployERC20Source(
 	return address, erc20Source
 }
 
+func RegisterERC20DestinationOnERC20Source(
```

Comment:
Minor, but IMO, it'd be cleaner to have a single `RegisterDestination` helper that doesn't check the reserve imbalance or multiplier settings. We could use a `iteleportertokensource.ITeleporterTokenSource` instance such that it works for all pairs of ERC20 and native sources and destinations.